manifests/0000_08_cluster-dns-operator_02-deployment.yaml

```diff
@@ -28,5 +28,5 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: OPERATOR_NAME
-              value: "cluster-dns-operator"
-      serviceAccountName: cluster-dns-operator
+              value: "openshift-dns-operator"
```

Comment:
The "openshift-" prefix stands out to me as different from what we did with the ingress operator.  Is the "openshift-" prefix there deliberately to distinguish the operator from kube-dns?

Reply:
I think it could just be `dns-operator` but I also don't think this affects anything but MAYBE log output?

Reply:
Actually, because we don't use `k8sutil.GetOperatorName()`, it doesn't affect anything.  We can go ahead and delete `OPERATOR_NAME` entirely.